utils/wasm-gen/src/generator/syscalls/invocator.rs

```diff
@@ -303,24 +300,31 @@ impl<'a, 'b> SysCallsInvocator<'a, 'b> {
         Ok(res)
     }
 
-    fn is_not_send_sys_call(&self, sys_call: InvocableSysCall) -> bool {
+    fn is_send_sys_call(&self, sys_call: InvocableSysCall) -> bool {
         use InvocableSysCall::*;
-        ![
+        [
             Loose(SysCallName::Send),
             Loose(SysCallName::SendWGas),
             Loose(SysCallName::SendInput),
             Loose(SysCallName::SendInputWGas),
             Precise(SysCallName::ReservationSend),
+            Precise(SysCallName::SendCommit),
+            Precise(SysCallName::SendCommitWGas),
         ]
         .contains(&sys_call)
     }
 
+    fn is_exit_sys_call(&self, sys_call: InvocableSysCall) -> bool {
+        matches!(sys_call, InvocableSysCall::Loose(SysCallName::Exit))
+    }
```

Comment:
can we introduce one function that will tell us to set/don't set the destination?

Reply:
done in 91134244f529aee6c23c3d44eb85d3a0e2c26999